doc/arch/001-screen-controllers.md

```diff
@@ -0,0 +1,102 @@
+# ADR: Screen Controllers
+
```

Comment:
Discussed and decided to not do because assumptions are not part of the ADR standard.

Reply:
Because ADRs are versioned, I think it'll be useful to link each other if we expect people to have existing context about something. 

For instance, when we write an ADR for explaining why `IN_FLIGHT` sync status was removed, will it not be useful to link to the PR that explains how sync works with these status enums?

Reply:
Yes we will link them by including a deprecation notice in the new ADR. So the status of the new ADR will say...

## Status 
Accepted
Deprecates 017-sync-status-enums.md